app/Resources/translations/default/messages.xlf

```diff
@@ -2960,4 +2945,205 @@ Comment: $this->list_modules_categories['search_filter']['name'] = $this->l('Sea
       </trans-unit>
     </body>
   </file>
+  <file original="modules/ps_emailsubscription/ps_emailsubscription.php" source-language="en-US" target-language="en" datatype="plaintext">
+    <body>
+      <trans-unit id="2df860f82be52de57fc5e3f7ecac9431">
+        <source>Newsletter subscription: Unable to delete customer using email.</source>
+        <target>Newsletter subscription: Unable to delete customer using email.</target>
+        <note>Line: 1267</note>
+      </trans-unit>
+      <trans-unit id="0b6664aa3bb3e3130368b120a8adfa91">
+        <source>Newsletter subscription: Unable to export customer using email.</source>
+        <target>Newsletter subscription: Unable to export customer using email.</target>
+        <note>Line: 1277</note>
+      </trans-unit>
+    </body>
+  </file>
+  <file original="modules/ps_themecusto/controllers/admin/AdminPsThemeCustoAdvanced.php" source-language="en-US" target-language="en" datatype="plaintext">
+    <body>
+      <trans-unit id="1bdd228669c9dd995eb664737e60786e">
+        <source>You do not have permission to edit this.</source>
+        <target>You do not have permission to edit this.</target>
+        <note>Line: 537</note>
+      </trans-unit>
+      <trans-unit id="c7e728f436eee2692d6c6f756621a70e">
+        <source>An error occured, please check your zip file</source>
```

Comment:
```suggestion
        <source>An error occurred, please check your zip file</source>
```

Reply:
But I think that this shouldn't be in the catalog, it has been removed from Crowdin since 1.7.4.

Reply:
Actually, all these keys from the Theme Custo module do not have to be here. I remember that it has been removed because after the 1.7.4 release because it came with its own translation. Anyway, as seen with @colinegin and @eternoendless lately, a small cleaning among the native modules needs to be done since there should not be all modules translatable via Crowdin.